versions/3.0.4.md

```diff
@@ -121,7 +121,9 @@ In order to preserve the ability to round-trip between YAML and JSON formats, YA
 
 An OpenAPI Description (OAD) MAY be made up of a single document or be divided into multiple, connected parts at the discretion of the author. In the latter case, [Reference Object](#reference-object) and [Path Item Object](#path-item-object) `$ref` keywords, as well as the [Link Object](#link-object) `operationRef` keyword, are used.
 
-Any document consisting entirely of an [OpenAPI Object](#openapi-object) is known as a **complete OpenAPI document**.
+Any document consisting entirely of an [OpenAPI Object](#openapi-object) is known as a **syntactically complete OpenAPI document**.
+An OpenAPI document that does ***not*** reference any other documents is known as a **self-contained OpenAPI document.***
```

Comment:
```suggestion
An OpenAPI document that does _not_ reference any other documents is known as a **self-contained OpenAPI document**.
```
Two changes:
- Three stars after "document" didn't pass markdown validation.
- Making "not" bold and italics would be a first in this spec, reduced it to italics.

Reply:
Thanks, @ralfhandl !